content/en/docs/ambient/usage/ztunnel/index.md

```diff
@@ -475,7 +457,7 @@ In the use cases so far, the traffic source and destination pods are both ambien
 
 ### East-West non-mesh pod to ambient mesh pod (and use of PeerAuthentication resource) {#ewnonmesh}
 
-In the example below, the same `httpbin` service which has already been set up in the prior examples is accessed via client `sleep` pods that are running in a separate namespace that is not part of the Istio mesh. This example shows that East-west traffic between ambient mesh pods and non mesh pods is seamlessly supported. Note that as described previously, this use case leverages the traffic hair-pinning capability of ambient. Since the non-mesh pods initiate traffic directly to the backend pods without going through HBONE or ztunnel, at the destination node, traffic is redirected via the ztunnel proxy at the destination node to ensure that ambient authorization policy is applied (this can be verified by viewing logs of the appropriate ztunnel proxy pod on the destination node; the logs are not shown in the example snippet below for simplicity).
+In the example below, the same `httpbin` service which has already been set up in the prior examples is accessed via client `sleep` pods that are running in a separate namespace that is not part of the Istio mesh. This example shows that East-west traffic between ambient mesh pods and non mesh pods is seamlessly supported. Since the non-mesh pods initiate traffic directly to the backend pods without going through HBONE or ztunnel. Note that in this case, you have control over whether or not the waypoint gets bypassed too depending on how you configure policy.
```

Comment:
I don't think user can control whether waypoint gets bypassed.  The traffic won't be sent to waypoint if it is from out of mesh.

Reply:
+1, I think it's just a wording/clarity issue though. You can control if bypass is allowed or denied using istio L4 policy but the way it is worded seems like it could mean that you can define a policy that makes the traffic hairpin to the waypoint which we do not offer.

Reply:
nits:

- no need to say 'Istio' 
- s/East-west/east-west
- s/through HBONE or ztunnel/through ztunnel  (we can elaborate on HBONE elsewhere)